src/webview_provider/newLineagePanel.ts

```diff
@@ -365,18 +367,19 @@ export class NewLineagePanel implements LineagePanelView {
     try {
       await Promise.all([
         ...Object.values(visibleTables).map(async (node) => {
-          let compiledSql: string | undefined;
-          if (node.config.materialized === "ephemeral") {
+          const nodeType = node.uniqueId.split(".")?.[0];
+          if (
+            !(nodeType === "model" || nodeType === "snapshot") ||
+            (nodeType === "model" && node.config.materialized === "ephemeral")
```

Comment:
added this because if we try to compile ephemeral models, we are getting the error that one of our users was also facing. for dbt-core 1.3, the error is: `Asked to compile class 'dbt.contracts.graph.compiled.CompiledModelNode' node, but it has no compiled form`

Reply:
also since the rest of the code deals with schema+compiled sql, by adding this condition we are skipping schema get for seed and source type of nodes